tasks/certificate.yml

```diff
@@ -22,5 +22,6 @@
     --cert-name '{{ certbot_nginx_cert_name }}'
     {% endif %}
     {% if letsencrypt_staging %} --staging {% endif %}
-  when: letsencrypt_cert.stat.exists and certbot_force_update is defined
+    {% if ansible_distribution_version >= "20.04" %} --non-interactive {% endif %}
+  when: letsencrypt_cert.stat.exists and certbot_force_update == true
```

Comment:
Not sure about the boolean conditional. It seems that you need to cast the variable to boolean first, like:
```
certbot_force_update | bool == True
```
https://www.shellhacks.com/ansible-when-variable-is-defined-exists-empty-true/

Reply:
Yes, and the condition is redundant, no?

we can use only:

```
certbot_force_update | bool
```